src/config/2019.json

```
@@ -746,6 +746,15 @@
       "avatar_url": "https:\/\/avatars2.githubusercontent.com\/u\/24447527?v=4&s=200",
       "github": "raghuramakrishnan71"
     },
+    "raghav": {
+      "name": "Raghvendra Kumar",
+      "teams": [
+        "developers"
+      ],
+      "avatar_url": "https:\/\/avatars3.githubusercontent.com\/u\/9475742?s=200&v=4",
```

Minor nit for consistency:

```suggestion
      "avatar_url": "https:\/\/avatars3.githubusercontent.com\/u\/9475742?v=4&s=200",
```